internal/tiltfile2/tiltfile_test.go

```diff
@@ -614,7 +614,24 @@ k8s_resource('foo', 'foo.yaml')
 }
 
 func TestDockerComposeResourceCreation(t *testing.T) {
-	// TODO(maia)
+	f := newFixture(t)
+	defer f.TearDown()
+
+	f.setupFoo()
+	f.file("docker-compose.yml", `
+version: '3'
+services:
+  foo:
+    build: ./foo
+    command: sleep 100
+    ports:
+      - "12312:12312"`)
+	YAMLPath := f.TempDirFixture.JoinPath("docker-compose.yml")
+	tf := fmt.Sprintf("docker_compose('%s')", YAMLPath)
+	f.file("Tiltfile", tf)
+
+	f.load("foo")
+	f.assertManifest("foo", dcYAMLPath(YAMLPath))
```

Comment:
nice. could you add a todo for me? -- assert Dockerfile and docker-compose.yml are config files